src/System.DirectoryServices.Protocols/tests/BerConverterTests.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.DirectoryServices.Protocols.Tests
+{
+    public class BerConverterTests
+    {
+        public static IEnumerable<object[]> Encode_TestData()
```

Comment:
I would have liked to share this data with Decode (to roundtrip), but the formats are different for encode and decode. Even when I adjust that, many bytes don't roundtrip